frontend/src/components/static_pages/about.jsx

```diff
@@ -169,4 +139,26 @@ class About extends Component {
   }
 }
 
-export default About;
+const mapStateToProps = state => {
+  return {
+    aboutKim: state.strapi.aboutKim,
+    aboutPierre: state.strapi.aboutPierre,
+    aboutShirley: state.strapi.aboutShirley,
+    kimPhoto: `http://localhost:1337${state.strapi.kimPhoto}`,
+    pierrePhoto: `http://localhost:1337${state.strapi.pierrePhoto}`,
+    shirleyPhoto: `http://localhost:1337${state.strapi.shirleyPhoto}`
```

Comment:
I believe you will need a forward slash `/` in between localhost:1337 and the state variables.